code/common/datastripe.q

```diff
@@ -4,25 +4,15 @@
 
 segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
 
-deletetablebefore:{![x;enlist (<;y;z);0b;0#`]}
 
-\d .
+tablekeycolsconfig:@[value;`.ds.tablekeycolsconfig;`tablekeycols.csv];    // getting the location of the tablekeycols.csv config file
 
-.rdb.datastripeendofperiod:{[currp;nextp;data]
-    .lg.o[`reload;"reload command has been called remotely"];
-    
-    // remove periods of data from tables
-    t:tables[`.] except .rdb.ignorelist;
-    lasttime:currp-.ds.periodstokeep*(nextp-currp);
-    tabs:{![x;enlist (<;y;z);0b;0#`]}'[t;`time;lasttime];
-    .lg.o[`reload;"Kept ",string[.ds.periodstokeep]," period",$[.ds.periodstokeep>1;"s";""]," of data from : ",", " sv string[tabs]];
+loadtablekeycols:{[]                                                          // loading the tablekeycols config as a dictionary
+    keypath:first .proc.getconfigfile[string .ds.tablekeycolsconfig];
+    @[{tablekeycols:(!/)(("SS";enlist",")0: hsym x)`tablename`keycol};keypath;{.lg.e[`init;"Failure in loading ",string y]}[;keypath]]
     };
 
+getstarttime:{[x] first x[`time]};
```

Comment:
Does this need to be min and max instead of first and last?

Reply:
Yes, it will need to be in case data arrives out of order.